src/Amp/Application.php

```diff
@@ -108,7 +108,7 @@ public function loadContainer() {
       $container->setParameter('instances_timeout', getenv('AMP_INSTANCES_TIMEOUT'));
     }
 
-    $container->setAlias('mysql', 'mysql.' . $container->getParameter('mysql_type'));
+    $container->setAlias('db', 'db.' . $container->getParameter('db_type'));
```

Comment:
There may be additional ways to do it (besides the pseudocode I posted). I just offered that as an example.